omod/src/main/java/org/openmrs/module/bedmanagement/rest/resource/BedDetailsResource.java

```diff
@@ -97,7 +97,7 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 	}
 	
 	@Override
-	public Object update(String uuid, SimpleObject propertiesToUpdate, RequestContext context) throws ResponseException {
+	public Object update(String bedId, SimpleObject propertiesToUpdate, RequestContext context) throws ResponseException {
```

Comment:
Well spotted. However obviously this is an API design flaw :-/

Reply:
Yes. Remind me if this is a new REST API with this Bahmni release? If so maybe I should fix the API now?

Reply:
I did a `git blame` and this is 5y old, so no worries I think.